enhancements/installer/vsphere-ipi-multiple-datacenter-clusters.md

```diff
@@ -0,0 +1,640 @@
+---
+title: multiple-datacenters-cluster-vsphere-ipi
+authors:
+  - "@jcpowermac"
+reviewers:
+  - "@rvanderp3"
+  - "@bostrt"
+  - "@JoelSpeed"
+  - "@staebler"
+  - "@lobziik"
+approvers:
+  - ""
+api-approvers:
+  - ""
+creation-date: 2021-09-21
+last-updated: 2022-03-11
+status: implementable
+see-also:
+  - "/enhancements/"
+replaces:
+  -
+superseded-by:
+  -
+---
+
+# Support Multiple vSphere vCenter Datacenters and Clusters in IPI
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [x] Design details are appropriately documented from clear requirements
+- [x] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+The goal of this enhancement is to provide the ability to install in a
+vSphere environment with multiple datacenters and clusters.
+
+This will be an opinionated design, the vCenter datacenter will always be a `region`
+and a vCenter cluster will always be a `zone`.
+
+## Motivation
+
+Users of OpenShift would like the ability to deploy
+within multiple datacenters and clusters to increase
+reliability. Customers would also like to take advantage
+of the concept of regions and zoning that this type of deployment would
+offer.
+
+- https://issues.redhat.com/browse/RFE-845
+- https://issues.redhat.com/browse/OCPPLAN-4927
+
+### Goals
+
+- Support regions and zones in vSphere using multiple datacenters (region) and
+clusters (zone)
+- Support installation into multiple datacenters and multiple clusters
+
+### Non-Goals
+
+- Support multiple subnets
+- Support multiple vcenters
+
+Note: The platform spec will be modified to support this at a future date.
+Only a single item in `vcenters` will be supported for the initial release.
+
+## Proposal
+
+Modification of the installer to support the provisioning of masters in defined
+datacenters and clusters.
+
+
+### api
+
+#### Infrastructure
```

Comment:
Added this section for the infrastructure `VSpherePlatformSpec`, this is a first pass at. 
@JoelSpeed 
@lobziik 
@rvanderp3 
@bostrt